dpti/hti.py

```diff
@@ -1386,6 +1388,50 @@ def hti_phase_trans_analyze(job, jdata=None):
     return if_phase_trans
 
 
+def run_task(task_dir, machine_file, task_name, no_dp=False):
+    job_work_dir_ = glob.glob(os.path.join(task_dir, task_name + "*"))
+    assert len(job_work_dir_) == 1
+    job_work_dir = job_work_dir_[0]
+    task_dir_list = glob.glob(os.path.join(job_work_dir, "task*"))
+    task_dir_list = sorted(task_dir_list)
+    work_base_dir = os.path.join(os.getcwd())
+    with open(machine_file) as f:
+        mdata = json.load(f)
+    machine = Machine.load_from_dict(mdata["machine"])
+    resources = Resources.load_from_dict(mdata["resources"])
+
+    submission = Submission(
+        work_base=work_base_dir,
+        resources=resources,
+        machine=machine,
+    )
+
+    if not no_dp:
+        task_list = [
+            Task(
+                command="ln -s ../../../graph.pb graph.pb; lmp -i in.lammps",
+                task_work_path=ii,
+                forward_files=["in.lammps", "conf.lmp"],
+                backward_files=["log*", "dump.hti", "out.lmp"],
+            )
+            for ii in task_dir_list
+        ]
+        submission.forward_common_files = ["graph.pb"]
+    else:
+        task_list = [
+            Task(
+                command="lmp -i in.lammps",
+                task_work_path=ii,
+                forward_files=["in.lammps", "conf.lmp"],
+                backward_files=["log*", "dump.hti", "out.lmp"],
+            )
+            for ii in task_dir_list
+        ]
```

Comment:
**Suggestion:** The `run_task` function seems to have repeated code blocks for creating `Task` objects with and without Deep Potential. This could be simplified by setting the command based on the `no_dp` flag and then creating the `Task` objects. [maintainability]
```suggestion
    command = "lmp -i in.lammps" if no_dp else "ln -s ../../../graph.pb graph.pb; lmp -i in.lammps"
    task_list = [
        Task(
            command=command,
            task_work_path=ii,
            forward_files=["in.lammps", "conf.lmp"],
            backward_files=["log*", "dump.hti", "out.lmp"],
        )
        for ii in task_dir_list
    ]
    if not no_dp:
        submission.forward_common_files = ["graph.pb"]
```

Reply:
This looks good?